db/hints/manager.hh

```diff
@@ -35,7 +35,7 @@
 #include <seastar/core/expiring_fifo.hh>
 #include "lister.hh"
 #include "gms/gossiper.hh"
-#include "locator/snitch_base.hh"
+#include "inet_address_vectors.hh"
```

Comment:
Why did you replace include instead of adding one?

Reply:
This might have been accidental.